starport-core/src/main/scala/com/krux/starport/db/tool/CreateUsers.scala

```diff
@@ -0,0 +1,85 @@
+package com.krux.starport.db.tool
+
+import com.krux.starport.config.StarportSettings
+import com.krux.starport.Logging
+import com.krux.starport.db.WaitForIt
+import com.krux.starport.db.table._
+import com.typesafe.config.ConfigValueFactory
+import scopt.OptionParser
+import slick.jdbc.PostgresProfile.api._
+import slick.jdbc.PostgresProfile.backend.DatabaseDef
+
+object CreateUsers extends App with Logging with WaitForIt with Schema {
+
+  lazy val starportSettings = StarportSettings()
+
+  case class Options(
+                      masterUser: String = "",
+                      masterPassword: String = "",
+                    ) {
+  }
+
+  def runQuery[T](query: DBIO[T], db: DatabaseDef): T = {
+    db.run(query).waitForResult
+  }
+
+  def createUser: DBIO[Int] = {
+    val userName = starportSettings.config.getString("krux.starport.jdbc.slick.properties.user")
+    val password = starportSettings.config.getString("krux.starport.jdbc.slick.properties.password")
+    logger.info(s"create role $userName")
+    sqlu"CREATE ROLE #$userName WITH LOGIN PASSWORD '#$password'"
```

Comment:
use the md5 version of creating the password `'md5[the-hash]'`

Reply:
from my testing this would be dependent on the value of `krux.starport.jdbc.slick.properties.password` and its up to the user to generate that value with md5:

```sh
$ echo -n abcdefkisuke | md5
d6e891f7e2ee0c83dd280e363f629a30
$ cat starport.conf
...
krux.starport.jdbc.slick.properties.password='md5d6e891f7e2ee0c83dd280e363f629a30'
...
$ psql -h 127.0.0.1 -d starport -U kisuke -W
Password for user kisuke: abcdef
```

Reply:
I meant we generate an md5 hash here from the password. But now I think about it, it's fine